testutils/utils.go

```diff
@@ -79,3 +81,29 @@ func ToJsonOrDie(value interface{}) []byte {
 		panic(err)
 	}
 }
+
+func WaitForAlertManagerSendGridReceiver(
```

Comment:
The name does not reflect what the util does. I will suggest naming it as: `WaitForAlertMangerSMTPReciverEmailConfigToUpdate`